mava/utils/logger.py

```diff
@@ -13,20 +13,19 @@
 # limitations under the License.
 
 import abc
-import json
 import logging
 import os
-import time
 import zipfile
 from datetime import datetime
 from enum import Enum
-from typing import Dict, List, Optional, Union
+from typing import Dict, List, Union
 
 import jax
 import neptune
 import numpy as np
 from colorama import Fore, Style
 from jax.typing import ArrayLike
+from marl_eval.json_tools import JsonWriter
```

Comment:
Then I'd say we should rename the mava `JsonLogger` to `MarlEvalLogger`

```suggestion
from marl_eval.json_tools import JsonLogger
```

Reply:
Done :)